syntax/make_commit.sublime-settings

```diff
@@ -0,0 +1,17 @@
+{
+	"always_show_minimap_viewport": false,
+	"auto_indent": false,
+	"detect_indentation": false,
+	"draw_indent_guides": false,
+	"draw_white_space": "None",
+	"font_options": [],
+	"gutter": false,
+	"line_numbers": false,
```

Comment:
Line numbers are great when  I am writing a commit message since I want to see that 2. line is empty and who many lines I wrote in the body.